clusters/app.ci/bookstack/bookstack.yaml

```diff
@@ -0,0 +1,222 @@
+          name: bookstack-db
+          ports:
+          - containerPort: 3306
+            name: mysql
+          env:
+          - name: MYSQL_DATABASE
+            value: "bookstack"
+          - name: MYSQL_PASSWORD
+            value: "secret"
+          - name: MYSQL_ROOT_PASSWORD
+            value: "secret"
+          - name: MYSQL_USER
+            value: "bookstack"
+          volumeMounts:
+          - mountPath: /var/lib/mysql
+            name: data
+        volumes:
+        - name: data
+          emptyDir:
+            sizeLimit: 500Mi
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: bookstack
+    namespace: ci
+    labels:
+      app: bookstack
+  spec:
+    selector:
+      app: bookstack
+    ports:
+    - port: 80
+      targetPort: 80
+      protocol: TCP
+    type: LoadBalancer
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: bookstack
+    namespace: ci
+    labels:
+      app: bookstack
+  spec:
+    replicas: 1
+    strategy:
+      type: Recreate
+    selector:
+      matchLabels:
+        app: bookstack
+    template:
+      metadata:
+        labels:
+          app: bookstack
+      spec:
+        containers:
+        - name: bookstack
+          image: docker.io/solidnerd/bookstack:22.11.0
+          ports:
+          - containerPort: 80
+            name: http
+          env:
+          - name: DB_DATABASE
+            value: "bookstack"
+          - name: DB_HOST
+            value: "bookstack-db:3306"
+          - name: DB_PASSWORD
+            value: "secret"
+          - name: DB_USERNAME
+            value: "bookstack"
+          volumeMounts:
+          - mountPath: /var/www/bookstack/.env
+            subPath: .env
+            name: dot-env
+          - mountPath: /var/www/bookstack/public/uploads
+            name: public-uploads
+          - mountPath: /var/www/bookstack/storage/uploads
+            name: storage-uploads
+          - mountPath: /var/www/bookstack/storage/clockwork
+            name: storage-clockwork
+        volumes:
+        - name: public-uploads
+          emptyDir:
+            sizeLimit: 100Mi
+        - name: storage-uploads
+          emptyDir:
+            sizeLimit: 100Mi
+        - name: storage-clockwork
+          emptyDir:
+            sizeLimit: 100Mi
+        - name: dot-env
+          secret:
+            secretName: dot-env
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: dot-env
+    namespace: ci
+  stringData:
+    .env: |
+      # Environment
+      APP_ENV=production
+      APP_DEBUG=${APP_DEBUG:-false}
+      APP_KEY=${APP_KEY:-SomeRandomStringWith32Characters}
+      # The below url has to be set if using social auth options
+      # or if you are not using BookStack at the root path of your domain.
+      APP_URL=${APP_URL:-null}
+      # Database details
+      DB_HOST=${DB_HOST:-localhost}
+      DB_DATABASE=${DB_DATABASE:-bookstack}
+      DB_USERNAME=${DB_USERNAME:-bookstack}
+      DB_PASSWORD=${DB_PASSWORD:-password}
```

Comment:
We can't store stuff like this in the git repo. We will have to create a secret in Vault, and then mirror it to `app.ci` via `ci-secret-bootstrap`. I think the content of this can stay the same in Vault, however. I will add this to Vault myself, as you won't have access, and then I will show you how to achieve the mirroring.